lib/headers.rb

```diff
@@ -0,0 +1,68 @@
+class Headers
+ 
+  def heading6(str)
+    if str.include?("###### ")
```

Comment:
`str.split(" ")[0].length` will dynamically tell you the length aka the header number